decidim-proposals/config/locales/en.yml

```diff
@@ -19,6 +19,11 @@ en:
         origin_feature_id: Component to copy the proposals from
   decidim:
     events:
+      comments:
+        proposal_mentioned:
+          email_intro: Your proposal "%{mentioned_proposal_title}" has been mentioned <a href="%{resource_path}">in this space</a> in the comments.
```

Comment:
Shouldn't`in this space` be the actual name/title of the commentable?

Reply:
@mrcasals and me agreed to use this "ambiguous" text to avoid having to switch case if the commentable has `title`, `name` or another attribute to reference it. Also to avoid having to remember to modify this switch/case when a new `commentable` exists in the system.